tools/SetupFlow/DevHome.SetupFlow/Windows/DevDriveWindow.xaml.cs

```diff
@@ -1,45 +1,16 @@
-// Copyright (c) Microsoft Corporation and Contributors.
-// Licensed under the MIT License.
+// Copyright (c) Microsoft Corporation and Contributors
+// Licensed under the MIT license.
 
-using DevHome.Common.Extensions;
-using DevHome.Common.Helpers;
+using DevHome.Common.Windows;
 using DevHome.SetupFlow.ViewModels;
 using DevHome.SetupFlow.Views;
-using Microsoft.UI.Xaml;
-using WinUIEx;
 
 namespace DevHome.SetupFlow.Windows;
-
-public sealed partial class DevDriveWindow : SecondaryWindowExtension
+public sealed partial class DevDriveWindow : SecondaryWindow
 {
-    private readonly DevDriveViewModel _devDriveViewModel;
-    private readonly DevDriveView _devDriveView;
-    private const double InitialHeight = 518;
-    private const double InitialWidth = 577;
-
     public DevDriveWindow(DevDriveViewModel viewModel)
-        : base()
-    {
-        _devDriveViewModel = viewModel;

```

Comment:
By extending `SecondaryWindow` we won't need to re-customize each secondary window.